WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewModel.swift

```diff
@@ -146,12 +146,21 @@ final class CardReaderSettingsConnectedViewModel: CardReaderSettingsPresentedVie
     }
 
     private func updateProperties() {
+        if let connectedReader = connectedReaders.first,
```

Comment:
I am wondering, is this the best place to disconnect the reader? After all, the purpose of this function is to update the properties with the first connected reader data; disconnecting the reader seems like an unexpected side effect here.

Reply:
Fair point: I could put it directly in the completion block for `observeConnectedReaders` in `beginObservation`. It won't make any difference to the functionality or way it works though: `updateProperties` is only called from that same block.

Reply:
Moved in 0a92d0f